CONTRIBUTING.md

```
@@ -0,0 +1,147 @@
+
+2. **Make your changes** and verify they pass:
+
+   ```bash
+   pytest
+   python -m ruff check --fix .
+   python -m ruff format .
+   ```
+
+3. **Commit** with a descriptive message using [conventional commit](https://www.conventionalcommits.org/) format:
+
+   ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)